clang/docs/ReleaseNotes.rst

```diff
@@ -115,7 +115,7 @@ C++ Language Changes
 - Allow single element access of GCC vector/ext_vector_type object to be
   constant expression. Supports the `V.xyzw` syntax and other tidbits
   as seen in OpenCL. Selecting multiple elements is left as a future work.
-- Implement `CWG1815 <https://wg21.link/CWG1815>`_. Support lifetime extension 
+- Implement `CWG1815 <https://wg21.link/CWG1815>`_. Support lifetime extension
```

Comment:
nit: generally we don't like such unrelated changes. We can land them seperately in other NFC patches.

Reply:
This is just stray whitespace cleanup that is automatically performed by most unix editors.

I don't think doing a whole separate commit to fix it is worth it, specially as there aren't a lot of lines changed here.

We do generally just ignore these unless there is so much noise it makes review difficult.

Reply:
Please don't do this in the future the policy is indeed to keep unrelated changes separate.

There are many practical reasons for this but mainly it reduces churn in the event of revert now we lose multiple changes and if they where unrelated that is just a waste all around.